WcaOnRails/app/assets/javascripts/users.js

```diff
@@ -96,13 +95,11 @@ onPage('users#index', function() {
     $('#search-box i').removeClass('fa-spinner fa-spin').addClass('fa-search');
 
     // Update params in the url.
-    var params = personsTableAjax.queryParams({
+    var params = usersTableAjax.queryParams({
```

Comment:
Ha! I have no idea how is it possible that `personsTableAjax` has been existing here. There's no such variable defined, whereas [there](https://github.com/jonatanklosko/worldcubeassociation.org/blob/f7d15d44eab597f8a094e302da35479c38979def/WcaOnRails/app/assets/javascripts/users.js#L52) is `usersTableAjax`, which makes much sense as we are in `users.js`. Actually if you check the users index on the website, it doesn't update the url properly at the moment.